src/api/spec/models/package_spec.rb

```diff
@@ -98,4 +104,74 @@
       end
     end
   end
+
+  context '#maintainers' do
+    before do
+      User.current = user
+    end
+
+    it 'returns an array with user objects to all maintainers for a package' do
+      # first of all, we add a user who is not a maintainer but a bugowner
+      # he/she should not be recognized by package.maintainers
+      package.add_user(other_user2, Role.find_by_title!("bugowner"))
+      package.save!
+
+      # we expect to get one user object in an array
+      package.add_user(user, maintainer_role)
```

Comment:
Instead of doing:
```ruby
package.add_user(user, maintainer_role)
package.save!
```

Just use the factory `relationship_package_user `:
```ruby
create(:relationship_package_user, user: user, package: package)
```